kong/router/expressions.lua

```diff
@@ -19,8 +24,34 @@ local PROTOCOLS_OVERRIDE = {
 }
 
 
-local function get_exp_and_priority(route)
+-- net.port => net.dst.port
+local function verify_expression(route)
```

Comment:
```suggestion
local function transform_expression(route)
```

Might be better. Verify should only return `true` or `false`.